src/logs_db/bot.py

```
@@ -71,7 +71,7 @@ def init_db():
     db_commit(query)
 
 
-def fetch_all(query, params=(), fetch_one=False):
+def fetch_all(query, params=[], fetch_one=False):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)